liquibase-dist/pom.xml

```diff
@@ -27,11 +27,12 @@
 		<db2.version>11.5.9.0</db2.version>
 		<firebird.version>5.0.4.java8</firebird.version>
 		<liquibase-pro.version>master-SNAPSHOT</liquibase-pro.version>
+		<liquibase-scripting.version>0-SNAPSHOT</liquibase-scripting.version>
```

Comment:
Should not be master-SNAPSHOT too?

Reply:
This is confusing to me, too.  The artifact that is being produced right now is `0-SNAPSHOT`.  I think that once devops is finished with their build work it should be `master-SNAPSHOT`.